runconfig/hostconfig.go

```diff
@@ -96,38 +109,46 @@ func (n PidMode) Valid() bool {
 	return true
 }
 
+// DeviceMapping represents the device mapping between the host and the container
 type DeviceMapping struct {
 	PathOnHost        string
 	PathInContainer   string
 	CgroupPermissions string
 }
 
+// RestartPolicy represents the restart policies of the container
 type RestartPolicy struct {
 	Name              string
 	MaximumRetryCount int
 }
 
+// IsNone indicates whether the container has no restart policy
```

Comment:
ohh right, I see what you mean. There is always a restartPolicy defined, but the default is no. I'll update.